proto/tendermint/abci/types.proto

```diff
@@ -124,6 +126,20 @@ message RequestApplySnapshotChunk {
   string sender = 3;
 }
 
+message RequestPrepareProposal {
+  // block_data is an array of transactions that will be included in a block,
+  // sent to the app for possible modifications.
+  // applications can not exceed the size of the data passed to it.
```

Comment:
```suggestion
  // applications can not return a list of transactions that exceeds `block_data_size`.
```
I am aware that the original comment probably conveys the semantics of your current implementation of `PrepareProposal`. I am adding this comment to raise awareness of the (possibly different semantics) in `v0.37.x`